TramsDataApi/Migrations/TramsDb/TramsDbContextModelSnapshot.cs

```diff
@@ -339,6 +339,35 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("AcademyTransferProjects","sdd");
                 });
 
+            modelBuilder.Entity("TramsDataApi.DatabaseModels.ProjectNote", b =>
+                {
+                    b.Property<int>("Id")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("int")
+                        .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);
+
+                    b.Property<int>("AcademyConversionProjectId")
+                        .HasColumnType("int");
+
+                    b.Property<string>("Author")
+                        .HasColumnType("nvarchar(max)");
+
+                    b.Property<DateTime>("Date")
+                        .HasColumnType("datetime2");
+
+                    b.Property<string>("Note")
+                        .HasColumnType("nvarchar(max)");
+
+                    b.Property<string>("Subject")
+                        .HasColumnType("nvarchar(max)");
+
+                    b.HasKey("Id");
+
+                    b.HasIndex("AcademyConversionProjectId");
+
+                    b.ToTable("ProjectNote","sdd");
```

Comment:
Could the name be something like `AcademyConversionProjectNotes` just so its clear they're project notes for our service